### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by Ellyboi**
**The requirement below was implemented:**
- Selected the API used.
- Build files structure for the React app.
- Prepare routes and navigation in the app.
- The user can display a list of items and filter them. Data came from the API.
- Created the tests for the application
- Styled the components.
- Deployed the project and test for final details.
- Recorded a video for the project.
- Created a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,234 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Making a Progressive Web App
+<!-- LIVE DEMO -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+## Live Demo <a name="live-demo"></a>
 
-### Advanced Configuration
+- [Live Demo](https://www.loom.com/share/df65b6d1292846c88af2370c5ea6f109)
```
Comment (Qoosim) [inline README.md]: - [ ] This link is supposed to link to your live project. Your deployed project. But it redirects to your presentation video. Kindly add your deployed link instead.

### Demo Link
demo-link

File: README.md [hunk 2]
```diff
@@ -1,70 +1,234 @@
...
+- npm test
+
+You will see the outcome in the console
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Deployment <a name="deployment"></a>
+
+To deploy this project in a local enviroment, you will have to run:
```
Comment (Qoosim) [inline README.md:161]: - [ ] It is required as part of project requirement to deploy your capstone project and add the link inside your README file. That requirement is yet to be implemented. Kindly do that.

File: README.md [hunk 3]
```diff
@@ -1,70 +1,234 @@
...
+- [ ] **[Creating new pages (about, contact, etc)]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- Project Presentation -->
+
+## :movie_camera: Project Presentation <a name="project-presentation"></a>
+
+- [Presentation](Country's Details App - 25 May 2023 - [https://www.loom.com/share/df65b6d1292846c88af2370c5ea6f109]
```
Comment (Qoosim) [inline README.md:196]: - [ ] This is another student presentation video inside your README file. How manage?. Kindly remove this and add your presentation video.

### Wrong Video
wrong-link

File: README.md [hunk 4]
```diff
@@ -1,70 +1,234 @@
...
+- Templete used in project provided [Nelson Sakwa](https://www.behance.net/sakwadesignstudio).
+- API used for the data [Air pollution](https://date.nager.at/api/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## License <a name="license"></a>
+
+This project is [MIT](./LICENSE(https://github.com/Ellyboi/react-capstone-project/blob/metricWebApp/LICENSE)) licensed.
```
Comment (Qoosim) [inline README.md]: - [ ] Kindly add a correct MIT License link to your README. It is broken at this moment.

license

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Ellyboi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Fair README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ellyboi** [comment]
Thanks for your time.....................correction made...

**Review by amejid — APPROVED** [review]
Hi @Ellyboi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
- React router used for navigation :heavy_check_mark: 
- Changes implemented :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**